docs/REST API/rest-api.md

```diff
@@ -3986,18 +3986,28 @@ List all configuration profiles for macOS hosts enrolled to Fleet's MDM that are
       "profile_id": 1337,
       "team_id": 0,
       "name": "Example profile",
+      "platform": "macOS",
```

Comment:
```suggestion
      "platform": "darwin",
```

In the API, for the `platform` property, we use `darwin` for macOS. This is more of a historical thing. I think we should stay consistent.

See the host vitals response as an example: https://fleetdm.com/docs/rest-api/rest-api#get-host